src/stacked_borrows.rs

```diff
@@ -329,7 +327,7 @@ impl<'tcx> Stack {
             // This pattern occurs a lot in the standard library: create a raw pointer, then also create a shared
             // reference and use that.
             // We *disable* instead of removing `Unique` to avoid "connecting" two neighbouring blocks of SRWs.
-            for idx in (granting_idx+1 .. self.borrows.len()).rev() {
+            for idx in (granting_idx + 1..self.borrows.len()).rev() {
```

Comment:
The spacing here is bad... looks like `1..self.borrows` is a thing. So there should be either no space, or parentheses.